spec/system/planning_applications/assessment_against_legislation_spec.rb

```diff
@@ -193,16 +188,18 @@
           "Comment added on 01 Sep 2022 by Alice Smith",
           with: ""
         )
-      end
 
-      within(row_with_content("D.1a")) do
         fill_in(
           "Comment added on 01 Sep 2022 by Alice Smith",
           with: "Updated comment"
         )
       end
 
       click_button("Save and come back later")
+
+    rescue Selenium::WebDriver::Error::UnexpectedAlertOpenError
+      visit planning_application_assessment_tasks_path(planning_application)
+
       click_link("Part 1, Class D")
 
```

Comment:
```suggestion
      visit planning_application_assessment_tasks_path(planning_application)

      begin
        click_link("Part 1, Class D")
      rescue Selenium::WebDriver::Error::UnexpectedAlertOpenError
      end
```

I think this is the only line that strictly needs to be wrapped.

Reply:
Oh, having said that, I've just gotten it to fail again after it passed ten times in a row.

Reply:
It's an absolute 🤯 

I don't think it's worth more of our time on this but appreciate you looking into it.
Either we just drop that part of the spec and do it as a new one or just keep that ambiguous ` rescue Selenium::WebDriver::Error::UnexpectedAlertOpenError` in.

What you reckon

Reply:
Either way let's not give this more time since it's not a priority

Reply:
I'm going to extract this into a new test